deployment/README.md

```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
 # semanage fcontext -a -t container_file_t "<STORAGE-PATH>(/.*)?"
 # restorecon -Rv <STORAGE-PATH>
 ```
+Add a new section to https://github.com/IBM/Db2/tree/develop/deployment.
+
+
+# How to create a new LDAP user
+## 1. Allocate the LDAP pod
+``` 
+oc get po | grep ldap | cut -d " " -f 1
+```
+Example output:
+```
+db2u-deployment-db2u-ldap-5cf745f7cf-ql6sh
+```
+## 2. Log in to the LDAP pod
+```
+oc rsh <LDAP-pod-name> /bin/bash
+```
+## 3. Find the next available LDAP user ID
+```
+source /opt/ibm/lib/utils.sh
+
+ldapPassword=$(get_ldap_root_password)
+
+echo $(($(ldapsearch -Z -H ldap:/// -D 'cn=bluldap,dc=blustratus,dc=com' -w "$ldapPassword" -b 'dc=blustratus,dc=com' '(objectClass=posixAccount)' | grep uidNumber | cut -d ":" -f 2 | sort -nr | head -n 1)+1)) 
+```
+Example output
+```
+5003
+```
+## 4. Set up a configuration file 
+Create a configuration file (e.g. */tmp/newuser.ldif*) with following contents:
+```  
+dn: uid=<user>,ou=People,dc=blustratus,dc=com
+uid: <user>
+cn: <user>
+objectClass: account
+objectClass: posixAccount
+objectClass: top
+uidNumber: <new LDAP uid>
+gidNumber: <3000|3002>
+homeDirectory: /mnt/blumeta0/home/<user>
+
+dn: cn=<bluadmin|bluusers>,ou=Groups,dc=blustratus,dc=com
+changetype: modify
+add: memberuid
+memberuid: <user>
+memberuid: uid=<user>,ou=People,dc=blustratus,dc=com   
+```
+**Note**: 
+* Replace *user* with a new LDAP user name. (e.g. ldapusr1)
+* gidNumber - **3000** means **admin**; **3002** means **user**
+* Replace *new LDAP uid* with an unused unique LDAP id from the step above. 
+
+## 5. Create the new LDAP user by using the config file
+Add the new user using the LDAP root credentials:
+```
+ldapadd -Z -H ldap:/// -D 'cn=bluldap,dc=blustratus,dc=com' -w "$ldapPassword" -f <config-file>
+```
+**Note**: 
+* Replace "config-file" with the real file that you created earlier (e.g. /tmp/newuser.ldif)
+
+Output:
+```
+adding new entry "uid=ldapuser1,ou=People,dc=blustratus,dc=com"
+modifying entry "cn=bluadmin,ou=Groups,dc=blustratus,dc=com"
+```
+
+## 6. Set the password for the newly created LDAP user
+```
+ldappasswd -x -Z -H ldap:/// -D "cn=bluldap,dc=blustratus,dc=com" -w "$ldapPassword" -S "uid=<user>,ou=People,dc=blustratus,dc=com" -s <password>
+```
+
+## 7. Verify the newly created LDAP user and credential
+a) Exit from the LDAP pod
+```
+exit
+```
+b) Log in to a db2u pod
+Example: 
```

Comment:
Can we skip "Example""? I don't think it is necessary

Reply:
Remove the word "Example"